web/src/app/projects/repository/repository.component.ts

```diff
@@ -66,6 +68,10 @@ export class RepositoryComponent implements OnInit, OnDestroy {
       .subscribe(() => setTimeout(() => this.manualReload = false, 60000)); // disable the ping button for 60 seconds;
   }
 
+  getRepoRating(): void {
```

Comment:
Already in `repository` component, so name should be fine as `getRating`.

But as there is no `return`, `get` seems odd, better to have something like `calculateRating`

Reply:
done